phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/apps/AppsController.java

```diff
@@ -113,4 +83,42 @@ public void setOnAppLoaded(Runnable onAppLoaded) {
     public ViewApps getView() {
         return view;
     }
+
+    private void populateView(List<CategoryDTO> categoryDTOS) {
+        Platform.runLater(() -> {
+            setDefaultCategoryIcons(categoryDTOS);
+            this.view.populate(categoryDTOS);
+        });
+    }
+
+    private void setDefaultCategoryIcons(List<CategoryDTO> categoryDTOS) {
+        try {
+            StringBuilder cssBuilder = new StringBuilder();
+            for (CategoryDTO category : categoryDTOS) {
+                cssBuilder.append("#" + category.getName().toLowerCase() + "Button{\n");
```

Comment:
Don't think so. The compiler should do that automatically (see [here](https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java)).